addons/spreadsheet_dashboard_sale_timesheet/data/files/timesheet_dashboard.json

```diff
@@ -1558,7 +1558,7 @@
             "colGroupBys": [],
             "context": {
                 "grid_anchor": "2022-09-12",
```

Comment:
what is this used for ? it looks wrong in the definition of a dashboard to have an hardcoded date...

Reply:
Coucou @VincentSchippefilt 🙂 

By default the grid view displays the current month, week depending on the default scale set on the view. The `grid_anchor` context allows to load the month/week containing the date set in that context instead of the current month/week. 🙂